src/activities/bargame/Bargame.qml

```diff
@@ -0,0 +1,666 @@
+/* GCompris - bargame.qml
+ *
+ * Copyright (C) 2016 UTKARSH TIWARI <iamutkarshtiwari@kde.org>
+ *
+ * Authors:
+ *   Yves Combe (GTK+ version)
+ *   UTKARSH TIWARI <iamutkarshtiwari@kde.org > (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+import GCompris 1.0
+
+import "../../core"
+import "bargame.js" as Activity
+
+ActivityBase {
+    id: activity
+    property int gameMode: 1
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: rootWindow
+        source: Activity.url + "school_bg" + bar.level + ".svg"
```

Comment:
There is an error at beginning (bar.level is still 0), can you handle it? Same for qrc:/gcompris/src/activities/bargame/Bargame.qml:96:9: QML Image: Cannot open: qrc:/gcompris/src/activities/bargame/resource/tux0.svg